CONTRIBUTING.md

```diff
@@ -169,18 +169,6 @@ you've signed the contribution.
 > `git ci -v` will get you a diff of your commit while writing your commit
 > message.
 
-> Note: If this is a non-code change, e.g. documentation, add `[doc only]` to the
-> PR subject line. This is to save CPU time on Travis CI, which lets us get more
-> build time for the other changes which actually change the code.
->
-> The tests actually run twice for each PR:
-> 
-> * when the PR is submitted for review
-> * when the PR is merged to the base branch
->
-> Having [doc only] in the commit skips the first one, but the merge commit also
-> needs it, so having it in the title (first line of commit) helps it easily
-> propagate to both places.
```

Comment:
Should we maybe support `[doc only]` in GitHub actions and skip tests if someone specifies it?
https://github.com/marketplace/actions/skip-based-on-commit-message

Reply:
I think we should wait until all functions are moved to GitHub Actions. I don't think we need for the doc only for the most cases. The path ignore will work in the most cases.

Reply:
What do you mean by `path ignore`? I'm newbie in GitHub Actions so don't know much yet.
I though that `[doc only]` would be useful if we are changing docs only and we don't want to run the whole build. It would save us some workers from unnecessary runs. That said, I'm fine dropping `[doc only]` support if it is complicated to add that feature right now. We may just open another task for this feature.

Reply:
GitHub actions can decide bases on the changes if it runs a workflow or not. For us the standard tests will never run if only things are changed inside of the docs directory.

Reply:
Oh cool. I didn't know about this feature. If so, you are right. We don't need `[doc only]`